assignment/assignment-impl/impl/src/java/org/sakaiproject/assignment/impl/OptionalFactoryBean.java

```diff
@@ -0,0 +1,36 @@
+package org.sakaiproject.assignment.impl;
```

Comment:
@ern Suggested moving this into kernel-util so that other parts of the codebase can use it.